products/almalinux.md

```diff
@@ -0,0 +1,24 @@
+---
+permalink: /almalinux
+layout: post
+title: AlmaLinux
+category: os
+command: lsb_release --release
+releasePolicyLink: https://blog.cloudlinux.com/announcing-open-sourced-community-driven-rhel-fork-by-cloudlinux
+activeSupportColumn: true
+releaseDateColumn: true
+sortReleasesBy: 'release'
+iconSlug: NA
+releases:
+  - releaseCycle: "8"
+    release: 2029-03-01
+    support: true
+    eol: false
```

Comment:
Can we put a release+10 year EOL date instead of `false`. Having a "mostly accurate" date (which might move a bit as it approaches) is much better than no date.

Reply:
Oops!  bad copy pasta.  Will fix